cmd/influxd/run/server_test.go

```diff
@@ -1010,6 +1010,9 @@ func NewConfig() *run.Config {
 	c.HTTPD.Enabled = true
 	c.HTTPD.BindAddress = "127.0.0.1:0"
 	c.HTTPD.LogEnabled = testing.Verbose()
+
+	c.UDP.BindAddress = "127.0.0.1:0"
+	c.UDP.Database = "db0"
```

Comment:
Well if I revert this change, tests still run and fail beacause there is no db nor bindaddress set